traitsui/qt4/tree_editor.py

```diff
@@ -1317,13 +1314,18 @@ def _on_nid_changed(self, nid, col):
             else:
                 self._set_label(nid, col)
 
-    def _menu_new_node(self, class_name, prompt=False):
+    def _menu_new_node(self, factory, prompt=False):
         """ Adds a new object to the current node.
         """
         node, object, nid = self._data
         self._data = None
-        new_node, new_class = self._node_for_class_name(class_name)
```

Comment:
`_node_for_class_name` is now no longer called anywhere.  Perhaps the method can be deleted?  It could be useful in some other case so maybe we want to just leave it, but given it is currently unused, it may make more sense to just delete it and re-add it if a future use case comes about that could make use of it.

Reply:
`_node_for_class_name` looks straightforward so if you don't find any users of it, please go ahead and remove it.